index.html

```diff
@@ -0,0 +1,274 @@
+                  src="./new-img/exhibition.svg"
+                  alt="exhibition"
+                />
+              </div>
+              <h3>Resilien</h3>
+              <p>
+                Resilience and Hope: Strength in Adversity a. Inspiring
+                photographs and stories showcasing the resilience and
+                determination of asylum seekers to rebuild their lives despite
+                the challenges they face.
+              </p>
+            </div>
+            <div class="program-content col-md-2">
+              <div class="image-wrap">
+                <img
+                  class="solo-content"
+                  src="./new-img/forum.svg"
+                  alt="lecture"
+                />
+              </div>
+              <h3>Debate</h3>
+              <p>
+                Cultural Exchange and Understanding: Participants engage in
+                activities promoting intercultural dialogue, breaking down
+                barriers, and fostering understanding between asylum seekers and
+                the host community.
+              </p>
+            </div>
+            <div class="program-content col-md-2">
+              <div class="image-wrap">
+                <img
+                  class="solo-content"
+                  src="./new-img/station.svg"
+                  alt="lecture"
+                />
+              </div>
+              <h3>Seminar</h3>
+              <p>
+                Each group will go through a series of simulated experiences,
+                such as navigating a complex bureaucratic process, facing
+                language barriers, or living in crowded conditions.
+              </p>
+            </div>
+            <div class="program-content col-md-2">
+              <div class="image-wrap">
+                <img
+                  class="solo-content"
+                  src="./new-img/ignite.svg"
+                  alt="lecture"
+                />
+              </div>
+              <h3>Kindled</h3>
+              <p>
+                attending ALD events offers the opportunity to network with ALD
+                affiliates from around the world, fostering ongoing connections
+                and potential collaborations beyond the duration of the summit.
+              </p>
+            </div>
+          </div>
+          <div class="see-program">
+            <a href="#">SEE THE WHOLE PROGRAM</a>
+          </div>
+          <div class="join-btn">
+            <button type="button" class="join btn btn-lg">Join as Now!</button>
+          </div>
+        </section>
+      </section>
+    </article>
+    <!-- Main Program Ends -->
+    <!-- Speakers Strats -->
+    <article id="featured-speakers" class="container featured-speakers">
+      <section class="speaker-intro">
+        <h2>Featured Speakers</h2>
+        <div class="speaker-intro-border"></div>
+      </section>
+      <div id="speaker-wrapper" class="row speaker-wrapper"></div>
```

Comment:
- [ ] Ouch! this is close but on the mobile version only the first 2 speakers are displayed as shown in the below Behance design and you're also forgetting the "More" button so please make sure to notice intricate details like this as this is one of the traits of a great frontend developer. 😉
- Behance design:
  ![image](https://github.com/Abbas-Aljuaifari/capstone-project-repo2/assets/104968236/e7b181dd-f63d-469a-a799-08d2bd739014)
- Your design:
  ![image](https://github.com/Abbas-Aljuaifari/capstone-project-repo2/assets/104968236/3995756d-b0c2-46ce-9373-901552d11eb6)
- [ ] **HINT:** You can simply achieve this by adding the below properties to your styles file:
```
  .speaker-wrapper > :nth-child(n+3){ 
     display: none;
   }
```